operator/internal/server/server.go

```diff
@@ -36,11 +36,11 @@ func NewWebServer(port string) *WebServer {
 	// Setup proxy data
 	token, err := ioutil.ReadFile(tokenFile)
 	if err != nil {
-		logger.V(3).Info("failed to read the token from kubernetes secrets")
+		logger.V(-2).Info("failed to read the token from kubernetes secrets")
```

Comment:
are there constants we could use instead of integers?

Reply:
this is how it showed this in their [docs](https://github.com/go-logr/logr#but-i-really-want-to-use-a-format-string)